docs/blog/arcade/general-release.md

```diff
@@ -0,0 +1,116 @@
+ 
+![New tilemap editor](/static/blog/arcade/general-release/tile-map-editor.png)
+
+If you do want to still use the old Tilemap blocks, you can add them as an extension called "Color-coded Tilemap".
+ 
+![Extension for old style tilemaps](/static/blog/arcade/general-release/old-tile-map.png)
+
+### Animation
+
+To make animations for your sprites, you can use the Animation extension. This is a collection of blocks that allow you to create different image frames of sprite motion – for example, you can create a Jumping or Walking sprite animation. 
+ 
+![Animation extension](/static/blog/arcade/general-release/animation-extension.png)
+
+![Animation editor](/static/blog/arcade/general-release/animation-editor.png)
+
+### Download to Hardware
+
+For those people who are lucky enough to have several different Arcade boards, we’ve made it easier to select and switch devices when downloading your game. 
+ 
+![Download to hardware](/static/blog/arcade/general-release/download-hardware.png)
+
+### Performance improvements
+
+We spent a lot of time working on making Arcade run faster, especially on Chromebooks and other lower-powered computers, so you should see snappier page loads and simulator runs now too!
+
+### GitHub authoring
+
+For games that are big/complex, or they have multiple people working on them, and for anyone writing Arcade extensions, we have a cool new integration with GitHub that allows you to store and work on your projects in a GitHub repo.   
+
+To bring in the full power of GitHub integration to the editor, a **GitHub** button is now present right next to the project **Save** button. Now you can have your project's changes synced to/from GitHub with just a button click.
+
+![GitHub button](/static/blog/arcade/general-release/github-integrate.png)
+
+Read more about this feature here – https://makecode.com/blog/makecode-with-github.
+
+## Future releases and updates
+
+Now that MakeCode Arcade has been released, we will be following the same annual update path we use for our other editors. This means we will only do one major update in the summer, so teachers can expect to have a stable version to work with during the school year.
+
+## Coding curriculum
+
+The MakeCode Arcade curriculum is freely available at https://arcade.makecode.com/courses/csintro. This curriculum covers Computer Science and game development concepts through a series of activities and lessons. Also, there are plenty of game tutorials, lessons, and samples available: https://arcade.makecode.com/docs. 
+ 
+![Lesson to choose from](/static/blog/arcade/general-release/lessons.png)
+
+## Connecting with us
+
+If you find any issues, or have suggestions, please log them on GitHub (https://github.com/microsoft/pxt-arcade/issues). If you have questions, or would like to participate in the MakeCode community, please join the Forum (https://forum.makecode.com) or follow us on social [@MSMakeCode](https://twitter.com/MSMakeCode).
```

Comment:
Is "social" a term now or say "social media"??

...follow us on social **media** [@MSMakeCode](https://twitter.com/MSMakeCode).

Reply:
"social media" is fine...